goplay/src/main/java/be/tapped/goplay/common/ktorClient.kt

```diff
@@ -0,0 +1,13 @@
+package be.tapped.goplay.common
+
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.apache.Apache
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.client.features.json.serializer.KotlinxSerializer
+
+internal val ktorClient: HttpClient = HttpClient(Apache) {
+    install(JsonFeature)
+    install(JsonFeature) {
+        serializer = KotlinxSerializer(jsonSerializer)
```

Comment:
```suggestion
        serializer = KotlinxSerializer(Json {
        isLenient = true
        ignoreUnknownKeys = true
    })
```

Inline the `internal val` and remove file? If we need to pass around `Json` as a dependency we should probably pass it around properly.

Reply:
I could not inline the `Json` since we need to re-use it for the manual JSON extraction from the HTML.  

https://github.com/timrijckaert/goplay-service/blob/replace-okhttp3-with-ktor/goplay/src/main/java/be/tapped/goplay/content/ProgramRepo.kt#L29